recipes/base_install.rb

```diff
@@ -116,11 +120,12 @@
   mode '0644'
 end
 
-if node['platform'] == 'ubuntu' && node['platform_version'].to_f >= 16.04
-  # FIXME: https://github.com/atomic-penguin/cookbook-nfs/issues/93
```

Comment:
This issue is still opened: https://github.com/atomic-penguin/cookbook-nfs/issues/93

Are we sure we can this workaround?

Reply:
On a second look this still seems to be an issue. Keeping original workflow for Ubuntu and adding workaround logic to skip `nfs::server4` for CentOS 8